index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home / Twitter</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <div class="grid-container">
+        <div class="left-sidebar">
+            <div class="leftbar twitterlogo"> <img src="/images/svg/twitter.svg"></div>
+            <div class="leftbar lefticon"><img src="/images/svg/home.svg"><p>Home</p></div>
+            <div class="leftbar lefticon"><img src="/images/svg/explore.svg"><p>Explore</p></div>
+            <div class="leftbar lefticon"><img src="/images/svg/notifications.svg"><p>Notifications</p></div>
+            <div class="leftbar lefticon"><img src="/images/svg/messages.svg"><p>Messages</p></div>
+            <div class="leftbar lefticon"><img src="/images/svg/bookmarks.svg"><p>Bookmarks</p></div>
+            <div class="leftbar lefticon"><img src="/images/svg/profile.svg"><p>Profile</p></div>
+            <div class="leftbar lefticon"><img src="/images/svg/more.svg"><p>More</p></div>
+            <div class="leftbar"><button class="tweetbar">Tweet</button></div>
+            <div class="leftbar empty"></div>         
+        </div>
+
+        <div class="main">
+            <div class="homemain">Home</div>
+            <div class="tweetuser">
+                <div class="profilepicture"><img src="images/images/userpfp.jpg"></div>
+                <div class="whatshappening">What's Happening?</div>
+                <div class="icontweet">
+                    <div class="media"><img src="/images/svg/media.svg"></div>
+                    <div class="gif"><img src="/images/svg/gif.svg"></div>
+                    <div class="stats"><img src="/images/svg/stats.svg"></div>
+                    <div class="emotes"><img src="/images/svg/emotes.svg"></div>
+                    <div class="calendar"><img src="/images/svg/calendar.svg"></div>
+                    <button>Tweet</button>
+                </div>
+
```

Comment:
I like that you spaced things out, even this 1 line makes it so much cleaner and easier to read, i definitely need to implement something like this, good luck to my reviewers haha